src/Run/CommandLine.cs

```diff
@@ -1813,7 +1825,7 @@ private static string GetEntryAssemblyName()
         private bool _noDashOnParameterSets = true;
         private bool _noSpaceOnQualifierValues;
         private string[] _parameterSetsWhereQualifiersMustBeFirst;
-        private bool _qualifiersUseOnlyDash = true;
+        private bool _qualifiersUseOnlyDash = false;
```

Comment:
it's the only way without a bunch of refactoring to support /.  I'm OK to undo it.